app/components/ActionLinks/index.js

```diff
@@ -53,54 +63,49 @@ const ExternalActionLinks = ({
         )}
 
         {isWhereabouts && whereaboutsUrl && (
-          <div className="action-links__link">
-            <ExternalActionLink
-              url={whereaboutsUrl}
-              image="/img/ICON_ManagePrisonerWhereabouts.png"
-              testId="whereabouts-link"
-            >
-              Manage prisoner whereabouts
-            </ExternalActionLink>
-          </div>
+          <ActionLink url={whereaboutsUrl} image="/img/ICON_ManagePrisonerWhereabouts.png" testId="whereabouts-link">
+            Manage prisoner whereabouts
+          </ActionLink>
         )}
 
-        {establishmentRollcheckUrl && (
-          <div className="action-links__link">
-            <ExternalActionLink
-              url={establishmentRollcheckUrl}
-              image="/img/EstablishmentRoll_icon.png"
-              testId="establishment-roll-link"
-            >
-              Establishment roll check
-            </ExternalActionLink>
-          </div>
+        {isEstablishmentRollCheck && establishmentRollcheckUrl && (
+          <ActionLink
+            url={establishmentRollcheckUrl}
+            image="/img/EstablishmentRoll_icon.png"
+            testId="establishment-roll-link"
+          >
+            Establishment roll check
+          </ActionLink>
         )}
 
         {hasAdminRights && adminUtilitiesUrl && (
-          <div className="action-links__link">
-            <ExternalActionLink
-              url={adminUtilitiesUrl}
-              image="/img/ICON_AdminUtilities.png"
-              testId="admin-utilities-link"
-            >
-              Admin and utilities
-            </ExternalActionLink>
-          </div>
+          <ActionLink url={adminUtilitiesUrl} image="/img/ICON_AdminUtilities.png" testId="admin-utilities-link">
+            Admin and utilities
+          </ActionLink>
+        )}
+
+        {isGlobalSearch && globalSearchUrl && (
```

Comment:
note this adds the global search task to everyone with the global search permission